study.md

```diff
@@ -39,7 +39,11 @@ should be symbols.
 | occupation | 'exterminator' |
 
 ```ruby
-# your answer here
+dale = {
```

Comment:
Grave accents don't work in Ruby like they do in JavaScript so this will cause an error.